storage/connect/frcas.h

```diff
@@ -1,117 +1,117 @@
-    case MSG_ACCESS_VIOLATN:  p = "Violation acc�s m�moire";                                                     break;
```

Comment:
We should have a comment at the top of this file with something like the following…

```
/* The non-ASCII literal strings in this file are encoded in the iso-8859-1 / latin-1 character set
 * (https://en.wikipedia.org/wiki/ISO/IEC_8859-1) */
```

And similar for other files with tables of literal strings.

Otherwise the intended encoding will continue to be non-obvious, and future developers may make mistakes like putting new UTF-8 encoded strings in here.